### ISSUE TITLE
Build documentation

### ISSUE THREAD

**Issue by codeknave** [synchrone/ivmetaserveremu#1]
Could you please provide a little info on how to build this? I have zero experience in Java so I'm not sure how to create the JAR file.

**Comment by account1203** [comment]
tl;dr no one cares, so probably the only option is making your own fork

I can't really help, but it seems there are issues with this project, like hardcoded paths with files not present here. Often code decay happens for such projects as well.

As the latest commit comment suggests, this project was imported from Google Code, where I can't find it anymore. It could have been developed in 2010 (e.g. see the date in defcon.sql)  and abandoned since then. I'm not even sure whether it was imported to GitHub by its original developer, or by some stranger.

The other, worse, issue, is that Introversion games have lost most of their players. Who would care to develop a private metaserver when games have to be scheduled elsewhere because there are hardly any active servers (not counting singleplayer games advertised by default)? And then some LAN emulation software like Tunngle or Hamachi would rather be used, if ports can't be forwarded for some reason.
